interfaces/builtin/accel.go

```diff
@@ -0,0 +1,56 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const accelSummary = `allows access to devices in the compute accelerators subsystem`
+
+const accelBaseDeclarationSlots = `
+  accel:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const accelConnectedPlugAppArmor = `
+# Description: Provide permissions for accessing devices and files of the compute accelerator subsystem
+
+# Access to accel character devices such as /dev/accel/accel0
+#
+# https://docs.kernel.org/accel/introduction.html
+#
+/dev/accel/accel* rw,
```

Comment:
On my system this is owned by the `render` group. The default user, at least on Fedora, is not a member of that group. This is just a statement, no action is necessary here.